frontend/src/lib/k8s/apiProxy.ts

```diff
@@ -985,7 +997,7 @@ export function patch(
   const opts = {
     method: 'PATCH',
     body,
-    headers: { ...JSON_HEADERS, 'Content-Type': 'application/json-patch+json' },
+    headers: { ...JSON_HEADERS, 'Content-Type': 'application/merge-patch+json' },
```

Comment:
Will this work for the other cases where it was working before? i.e. did you check just changing e.g. a label for a resource?

Reply:
Yupp i think in this is just about patching what we send otherwise we would have to add json operations as well if we use the older way which i am not sure if is supported